packages/web/components/MegaMenu/MegaMenuHeader.tsx

```diff
@@ -211,25 +210,20 @@ const SearchModal = ({
         ),
         shareReplay(1),
       )
-      .subscribe(([p, g]) => {
-        setSearchResults({ players: p.players, guilds: g.guilds });
+      .subscribe(([{ players: p }, { guilds: g }]) => {
+        setSearchResults({ players: p, guilds: g });
```

Comment:
Why rename those variables when it's shorter not to?

Are you worried about shadowing the values from above in the component? Do you get a linting error when you do so?

```suggestion
      .subscribe(([{ players }, { guilds }]) => {
        setSearchResults({ players, guilds });
```

Reply:
Yes I was getting lint error